src/__tests__/redux/actions/signup.js

```diff
@@ -0,0 +1,51 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import mockAxios from 'axios';
+import { signupUser, signupFailure, signupSuccess } from '../../../redux/actions/authActions';
+
+const mockStore = configureMockStore([thunk]);
+
+const mockData = {
+  name: 'vidar-name',
+  username: 'vidar-username',
+  email: 'vidar-email',
+};
+
+describe('<SIGNUP ACTION CREATORS />', () => {
```

Comment:
@kleva-j why is this describe written this way?

Reply:
Now working on this.